src/tools/cephfs/cephfs_shell.py

```diff
@@ -0,0 +1,848 @@
+                path = ''
+                if args.parent :
+                    files = reversed(sorted(set(dirwalk(cephfs.getcwd().decode('utf-8') + dir_path))))     
+                    for i, path in enumerate(files):
+                        if path[1:] != dir_path : 
+                            print('Removed : ', path)
+                            try :
+                                cephfs.rmdir(to_bytes(path))
+                            except :
+                                cephfs.unlink(to_bytes(path)) 
+                if not is_pattern and dir_path != path:                      
+                    cephfs.rmdir(to_bytes(dir_path))
+        except :
+            traceback.print_exc(file = sys.stdout)
+    
+    def complete_rm(self, text, line, begidx, endidx):
+        """ 
+        auto complete of file name.
+        """ 
+        return self.complete_filenames(text, line, begidx, endidx)
+
+    rm_parser = argparse.ArgumentParser(description = 'Remove File.')
+    rm_parser.add_argument('file_paths', help = 'File Path.', nargs = '+')
+
+    @with_argparser(rm_parser)
+    def do_rm(self, args):
+        """
+        Remove a specific file
+            Usage : rm <file_path>
+        """
+        try:
+            files = args.file_paths
+            print(args.file_paths)
+            for file_path in files :
+                if file_path.count('*') > 0 :
+                    files.extend(get_all_possible_paths(file_path))
+                else :
+                    cephfs.unlink(to_bytes(file_path))
+        except :
+            traceback.print_exc(file = sys.stdout)
+   
+    def complete_mv(self, text, line, begidx, endidx):
+        """
+         auto complete of file name.
+        """    
+        return self.complete_filenames(text, line, begidx, endidx)
+
+    mv_parser = argparse.ArgumentParser(description = 'Move File.')
+    mv_parser.add_argument('src_path', type = str, help = 'Source File Path.')
+    mv_parser.add_argument('dest_path', type = str, help = 'Destination File Path.')
+
+    @with_argparser(mv_parser)
+    def do_mv(self, args):
+        """
+        Rename a file or Move a file from source path to the destination
+            Usage : mv <src_path> <dest_path>
+        """
+        try:           
+            cephfs.rename(to_bytes(args.src_path), to_bytes(args.dest_path))
+        except :
+            traceback.print_exc(file = sys.stdout)
+   
+    def complete_cd(self, text, line, begidx, endidx):
+        """ 
+        auto complete of file name.
+        """    
+        return self.complete_filenames(text, line, begidx, endidx)
+
+    cd_parser = argparse.ArgumentParser(description = 'Create Directory.')
+    cd_parser.add_argument('dir_name', type = str, help = 'Name of the directory.', nargs = '*', default = [''])
+
+    @with_argparser(cd_parser)
+    def do_cd(self, args):
+        """ 
+        Open a specific directory.
+            Usage : cd <dir_name>
```

Comment:
`cd [dir_name]`

Closed brackets indicate the argument is optional. Make sure the rest of the documentation is consistent.